norddrop/src/device.rs

```diff
@@ -528,39 +477,39 @@ impl NordDropFFI {
         }
 
         for desc in descriptors {
-            if let Some(content_uri) = &desc.content_uri {
-                #[cfg(target_os = "windows")]
-                {
-                    let _ = content_uri;
+            match desc {
```

Comment:
Just checking in that `cfg(target_os = "windows")` is missing, is that intended?

Reply:
There's `cfg(unix)` instead, It's (almost) the same thing but reversed and more concise